browser/src/neovim/NeovimInstance.ts

```diff
@@ -718,7 +739,7 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
 
             this._onMessage.dispatch({
                 severity: "error",
-                title: "Problem loading `init.vim`:",
+                title: "Nvim startup warning:",
```

Comment:
I swapped this as my understanding is that this catches any startup warnings, which could be anything, not just init.vim errors.